README.md

```diff
@@ -83,6 +84,8 @@ dotnet DocLinkChecker -v -f .github/workflows/DocLinkChecker.config
 
 If the above command fails because "you must install or update .NET", follow the URL from the failed command's output or [this one](https://dotnet.microsoft.com/en-us/download/dotnet/6.0/runtime?cid=getdotnetcore&os=windows&arch=x64) to download and install .NET runtime 6. Dotnet supports simultaneous installation of several .NET runtime versions, and version 6 is required to run the DocLinkChecker.
 
+The above set of commands can also be run using the `docfx-util.ps1` Powershell script. Specifically, run `./docfx-util.ps1 -d` in the repo's root directory. 
```

Comment:
This is good, but I think we should add a section towards the top that is just a short description of the utility script and a list of the commands / what they do. I know where to look and I almost missed which commands I needed to run for each level of the script. This is not to say that they should be removed from where they are, I think that it makes sense to have a brief snippet after describing the underlying commands that it is already implemented in the script.

Reply:
One important comment with this command (and also with `-a`) is that it does not serve the website, which I think is correct. But we should add a line stating that to quickly view the built website, they can just run `dotnet docfx --serve`. Possibly redundant, since we specify this elsewhere I believe, but not in the specific context of the `./docfx-util.ps1`  commands.